keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Job success/completion policy
+kep-number: 3998
+authors:
+  - "@tenzen-y"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+status: implementable
+creation-date: 2023-06-05
+reviewers:
+  - "@mimowo"
+  - "@alculquicondor"
+approvers:
+  - "@wojtek-t"
+  - "@soltysh"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.29"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.29"
+  beta: "v1.30"
+  stable:
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: JobSuccessPolicy
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - jobs_finished_total
```

Comment:
I think this section means metrics that this feature is adding. You probably want metrics for this feature but I think these are existing metrics.

Reply:
SGTM Thanks!

Reply:
Done.